File: app/src/main/java/com/grace/placessearch/data/model/Meta.java
package com.grace.placessearch.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Meta {

    @SerializedName("code")
    @Expose
    private long code;
    @SerializedName("requestId")
    @Expose
    private String requestId;

    public long getCode() {
        return code;
    }

    public void setCode(long code) {
        this.code = code;
    }

    public String getRequestId() {
        return requestId;
    }

    public void setRequestId(String requestId) {
        this.requestId = requestId;
    }

}

File: app/src/main/kotlin/com/grace/placessearch/common/data/model/Meta.kt
package com.grace.placessearch.common.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Meta {

    @SerializedName("code")
    @Expose
    var code: Long = 0
    @SerializedName("requestId")
    @Expose
    var requestId: String? = null

}